src/vs/base/browser/ui/resourceviewer/resourceViewer.ts

```diff
@@ -14,6 +14,7 @@ import { Builder, $ } from 'vs/base/browser/builder';
 import DOM = require('vs/base/browser/dom');
 import { DomScrollableElement } from 'vs/base/browser/ui/scrollbar/scrollableElement';
 import { BoundedLinkedMap } from 'vs/base/common/map';
+import { IWindowsService } from 'vs/platform/windows/common/windows';
```

Comment:
@mjbvz we try to have base not depending on platform if possible (see https://github.com/Microsoft/vscode/wiki/Code-Organization#layers)